server/routes/help/content/aboutPatientsHelp.ts

```diff
@@ -1,11 +1,15 @@
-import type { HelpSection } from './index'
+// eslint-disable-next-line import/no-cycle
```

Comment:
There a class that uses a type that's defined in the same file and I get all these no-cycle warning that I don't really understand.